xcube_clms/store.py

```diff
@@ -98,3 +142,12 @@ def get_search_params_schema(
         cls, data_type: DataTypeLike = None
     ) -> JsonObjectSchema:
         pass
+
+    def preload_data(self, *data_ids: str, **preload_params):
```

Comment:
Missing preload handle. See related xcube issue.

Reply:
Will be handled in #10